### ISSUE TITLE
[Adapter Cards-Schema Explorer]: Tool tip is missing for the Hamburger menu icon in the 400% Settings.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7852]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)](https://adaptivecards.io/explorer/)](https://adaptivecards.io/)

Pre-Requisites:
Change Display Resolution to 1024* 768.
Zoom the page to 400%

Repro Steps:
1.Open above URL with V- credential.
2.Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
3.Navigate to 'All Microsoft' link using tab key and observe the issue.

Actual Result:
There is no Tool tip for the Hamburger Menu icon in 400% settings.

Expected Result:
There should be a tool tip for the Hamburger Menu icon like "All Microsoft" Menu.

Observation:
Same issue exists in 200% settings also.

Note:
**Path:** Schema Explorer-->Media(Card Elements)-->Adaptive Card video.
 Tool tip is missing for Play and forward and Full screen and mute buttons.
PFA: Tool tip is missing on video buttons.mp4

User Experience:
It will impact the low vision users. if tool tip is not present on the Hamburger Menu icon, users will get confused, if alt text label is not given for this menu icon.

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.03

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="960" alt="Tool tip is missing for the Hamburger Menu icon" src="https://user-images.githubusercontent.com/104482612/190155637-ec080f08-a367-4248-b3b3-42ab91c51952.png">

![Tooltipof hamburger](https://user-images.githubusercontent.com/104482612/191251954-aba86c4a-12dc-4d57-ab00-ff4577e52d29.png)

https://user-images.githubusercontent.com/104482612/191484378-b7d9979d-1a40-4d9d-9c21-2b2ffd9dce71.mp4

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by vagpt** [comment]
@anna-dingler,

Could you please let us know do you have any plan to fix this issue?

As, issue is still repro.

**Comment by vagpt** [comment]
Same issue is repro at 1280*1024 resolution as well.

![image](https://github.com/user-attachments/assets/3b88ba7c-1bd2-4aac-b1d9-cbd01db2dd90)
